README.adoc

```diff
@@ -1,9 +1,13 @@
 This repo contains utilities that use Jenkins for early warnings/error detection in jiras and other jira tools.
 
-README     	     this file
-render-testcase.py   takes a report.json to render jira query results into junit.xml test reports   
-*.json               reports used by JBoss Tools team for detecting stale/non-triaged/incorrect jiras
-bzira.py             utility to make bugs returned by bugzilla query show up in jira to use in sprint-planning.
+== Contents of this repository 
+- *README*: this file
```

Comment:
I'd argue this doesn't need explaining but :man_shrugging:

Reply:
:rofl: